engine/glfw/lib/android/android_window.c

```diff
@@ -712,7 +712,7 @@ Java_com_dynamo_android_DefoldActivity_nativeOnActivityResult(
     JNIEnv *env, jobject thiz, jobject activity, jint requestCode,
     jint resultCode, jobject data) {
 
-    for (int i = 0; i < g_ListenersCount; ++i)
+    for (int i = g_ListenersCount - 1; i >= 0 ; --i)
```

Comment:
If developer try to unregister lister from listener function itself, it breaks logic for the other registered listeners. One of them don't trigger.

Since we don't guarantee the order how we trigger these listeners and in most of the cases of using `OnCreate` it's a good practice to unregister listener from listener itself  I think this iterator will be better.

Reply:
Good find!
Another benefit is also that we get the reverse-order mechanic.

Reply:
yes, @ekharkunov-ms found it!